demos/demo_finger_torque_control.cpp

```diff
@@ -30,23 +30,24 @@ struct Hardware
     std::shared_ptr<Sliders<3>> sliders;
 };
 
-static THREAD_FUNCTION_RETURN_TYPE control_loop(void *hardware_ptr)
-{
-    // cast input arguments to the right format --------------------------------
-    Hardware &hardware = *(static_cast<Hardware *>(hardware_ptr));
+// static THREAD_FUNCTION_RETURN_TYPE control_loop(void *hardware_ptr)
+// {
+//     // cast input arguments to the right format --------------------------------
+//     Hardware &hardware = *(static_cast<Hardware *>(hardware_ptr));
 
-    // position controller -----------------------------------------------------
-    while (true)
-    {
-        TimeIndex t = hardware.finger->append_desired_action(
-            hardware.sliders->get_positions());
+//     // position controller -----------------------------------------------------
+//     while (true)
+//     {
+//         TimeIndex t = hardware.finger->append_desired_action(
+//             hardware.sliders->get_positions());
 
-        hardware.finger->get_observation(t);
-    }
-    return THREAD_FUNCTION_RETURN_VALUE;
-}
+//         hardware.finger->get_observation(t);
+//     }
+//     return THREAD_FUNCTION_RETURN_VALUE;
+// }
```

Comment:
why did you comment this out?

Reply:
for warning removal, we need to fix or remove the demo.